### PR TITLE
BaSP-TG-52: Standarize errors code 

### PR DESCRIPTION
**Pull request by GonzaloAsencio**
Change "data.msg" to "data.message" in "index.js" on Project folder.

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (Ignacio-Sormani) [inline .vs/VSWorkspaceState.json:1]: Don't push files under the .vs folder

File: src/Components/Projects/Form/index.js [hunk 2]
```diff
@@ -99,10 +99,10 @@ const index = () => {
       const data = await response.json();
 
       if (response.status === 200) {
```
Comment (Ignacio-Sormani) [inline src/Components/Projects/Form/index.js:101]: It was a PUT method, so succesful response status should be 201

File: src/Components/Projects/List/index.js [hunk 3]
```diff
@@ -25,9 +25,9 @@ const ProjectList = ({ projectItem, onDeleteItem }) => {
 
     if (response.status === 200) {
```
Comment (Ignacio-Sormani) [inline src/Components/Projects/List/index.js:26]: It was a DELETE method, so succesful response status should be 204

**Review by Ignacio-Sormani — CHANGES_REQUESTED** [review]

**Review by Ignacio-Sormani — APPROVED** [review]

**Review by MaicolFreire001 — APPROVED** [review]

File: src/Components/Projects/Form/index.js [hunk 4]
```diff
@@ -96,11 +96,11 @@ const index = (props) => {
...
-        alert(data.msg);
+      if (response.status === 201) {
+        alert(data.message);
         props.history.goBack();
       } else if ([404, 500].includes(response.status)) {
-        alert(data.msg);
+        alert(data.message);
       } else if (response.status === 400) {
         alert(data.message);
       }
```
Comment (fede230298) [inline src/Components/Projects/Form/index.js:106]: After this change, we could just make it into one else if.

File: src/Components/Projects/List/index.js [hunk 5]
```diff
@@ -24,11 +24,11 @@ const ProjectList = ({ projectItem, onDeleteItem }) => {
...
+    if (response.status === 204) {
       onDeleteItem(projectItem._id);
-      alert(data.msg);
+      alert(data.message);
     } else if ([404, 500].includes(response.status)) {
-      alert(data.msg);
+      alert(data.message);
     } else if (response.status === 400) {
       alert(data.message);
     }
```
Comment (fede230298) [inline src/Components/Projects/List/index.js:34]: Same as above

**Review by fede230298 — APPROVED** [review]

**Pull request merged**